charts/osm/templates/osm-injector-deployment.yaml

```diff
@@ -94,6 +94,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: OSM_ENVOY_IMAGE
+              value: "{{ .Values.osm.sidecarImage }}"
+            - name: OSM_ENVOY_WINDOWS_IMAGE
+              value: "{{ .Values.osm.sidecarWindowsImage }}"
+            - name: OSM_INIT_CONTAINER_IMAGE
+              value: '{{ include "osmSidecarInit.image" . }}'
```

Comment:
I would include `DEFAULT` somewhere in these names to make it clearer that these are fallbacks and not authoritative.